test/parallel/test-dgram-multicast-setIf.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const socket = dgram.createSocket('udp4');
+const socket2 = dgram.createSocket('udp4');
```

Comment:
Instead of having `socket`, `socket2`, ..., `socketN`, we've been moving toward using block scoping to isolate tests. So:

```javascript
{
  const socket = dgram.createSocket('udp4');
  ...
}

{
  const socket = dgram.createSocket('udp4');
  ...
}
```